.github/workflows/go.yml

```
@@ -0,0 +1,29 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.12
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.12
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Get dependencies
+      run: go mod download
+
+    - name: Ensure the project is formatted
+      run: test -z "$(go fmt ./...)"
+
+    - name: Analyse Go code using `vet`
+      run: go vet ./...
+
+    - name: Test
+      run: go test
```

Could you change this to use [Ginkgo](https://onsi.github.io/ginkgo/)?

```suggestion
      run: go run github.com/onsi/ginkgo/ginkgo -r -requireSuite --randomizeAllSpecs --randomizeSuites --failOnPending --cover --trace --race --compilers=2
```

---

will do